modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds Fetchpriority parameter to the none lazy loaded images.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since 1.5.0
```

Comment:
Should use `n.e.x.t`.

```suggestion
 * @since n.e.x.t
 * @package performance-lab
```